easybuild/easyconfigs/l/libxc/libxc-4.0.3-foss-2017a.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'libxc'
+version = '4.0.3'
+
+homepage = 'http://www.tddft.org/programs/octopus/wiki/index.php/Libxc'
+description = """Libxc is a library of exchange-correlation functionals for density-functional theory.
+ The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+# Results for some functionals (e.g. mgga_c_tpss) deviate with too aggressive optimization settings.
+toolchainopts = {'lowopt': True}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://www.tddft.org/programs/octopus/down.php?file=libxc/4.0.3/']
```

Comment:
@pouillon Sorry, I missed this earlier: please use `%(version)s` here rather than hardcoding the version, see also https://github.com/easybuilders/easybuild-easyconfigs/pull/5258/files#diff-37a43d0bbaf440e0fd23c9b86fa9a99dR15 .

Same remark for your other `libxc` pull requests (btw, don't worry to pool PRs like that together in a single PR in the future, that creates less overhead for us; a single PR for different version/toolchains of one particular software package is fine, as long as it's manageable (rule of thumb: < 10 easyconfigs)).

Reply:
I had initially put %(version)s in the URL but it failed, which made me switch back to hard-coded number. I just tried again and this time it worked. I probably had mistyped it the first time and hadn't realized it. Much better now.

I take good note of the PR grouping advice, this is also less overhead for me.